core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/quiverPlot/QuiverPlotOpDesc.scala

```diff
@@ -0,0 +1,135 @@
+package edu.uci.ics.texera.workflow.operators.visualization.quiverPlot
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import com.kjetland.jackson.jsonSchema.annotations.{JsonSchemaInject, JsonSchemaTitle}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{Attribute, AttributeType, Schema}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.operators.visualization.{
+  VisualizationConstants,
+  VisualizationOperator
+}
+
+@JsonSchemaInject(json = """
+{
+  "attributeTypeRules": {
+    "value": {
+      "enum": ["integer", "long", "double"]
+    }
+  }
+}
+""")
+class QuiverPlotOpDesc extends VisualizationOperator with PythonOperatorDescriptor {
+
+  //property panel variable: 4 requires: {x,y,u,v}, 2 optional: {x,y} for specific points
+
+  @JsonProperty(value = "x", required = true)
+  @JsonSchemaTitle("x")
+  @JsonPropertyDescription("column for the x-coordinate of the starting point")
+  @AutofillAttributeName var x: String = ""
+
+  @JsonProperty(value = "y", required = true)
+  @JsonSchemaTitle("y")
+  @JsonPropertyDescription("column for the y-coordinate of the starting point")
+  @AutofillAttributeName var y: String = ""
+
+  @JsonProperty(value = "u", required = true)
+  @JsonSchemaTitle("u")
+  @JsonPropertyDescription("column for the vector component in the x-direction")
+  @AutofillAttributeName var u: String = ""
+
+  @JsonProperty(value = "v", required = true)
+  @JsonSchemaTitle("v")
+  @JsonPropertyDescription("column for the vector component in the y-direction")
+  @AutofillAttributeName var v: String = ""
+
+  //The list to holds all the x coordinates of the user's input
+  @JsonProperty(value = "pointsX", required = false)
+  @JsonSchemaTitle("x coordinates")
+  @JsonPropertyDescription("x coordinates of points")
+  var pointsX: java.util.List[java.lang.Double] = new java.util.ArrayList[java.lang.Double]()
```

Comment:
let's remove the codes about drawing points. it is confusing when users use this operator